src/devices/Max7219/Register.cs

```diff
@@ -24,4 +24,4 @@ public enum Register : byte
         SHUTDOWN = 0xC,
         DISPLAYTEST = 0xF
     }
-}
```

Comment:
since you already are onto it, this class can be private I presume

Reply:
This code was not written by me. I changed the type to internal, but many public methods called it, so I gave up changing it. Maybe it was my VS saved the change, and I forgot to discard it.